src/icons/fileIcons.ts

```diff
@@ -1720,6 +1720,10 @@ export const fileIcons: FileIcons = {
         'mcworld',
         'mine',
         'mus',
+        'mcstructure',
+        '.mcattributes',
```

Comment:
Thanks for this PR! I was wondering if ".mcattributes", ".mcdefinitions" and ".mcignore" are file extensions or file names? If these are file extensions you've to remove the dot before the extension name "mcattributes". If you meant file names, you have to put these names into a separate property of this icon which is called `fileNames`. It's on the same level as `fileExtensions`. You can take a look at other icon definitions in this file for reference :)

Reply:
Oh, its fileNames, I didn't notice that! Fixed.

Reply:
No problem, looks fine now :) Thanks!